Disable test_test_protocol2, which depends on testtools own test suite.

--- python-subunit-1.4.3/python/subunit/tests/__init__.py.orig	2023-09-17 14:40:32.000000000 +0200
+++ python-subunit-1.4.3/python/subunit/tests/__init__.py	2023-09-29 06:12:29.112435057 +0200
@@ -31,7 +31,7 @@ from subunit.tests import (test_chunked,
                            test_progress_model, test_run, test_subunit_filter,
                            test_subunit_stats, test_subunit_tags,
                            test_tap2subunit, test_test_protocol,
-                           test_test_protocol2, test_test_results)
+                           test_test_results)
 
 
 def test_suite():
@@ -42,7 +42,6 @@ def test_suite():
     result.addTest(loader.loadTestsFromModule(test_progress_model))
     result.addTest(loader.loadTestsFromModule(test_test_results))
     result.addTest(loader.loadTestsFromModule(test_test_protocol))
-    result.addTest(loader.loadTestsFromModule(test_test_protocol2))
     result.addTest(loader.loadTestsFromModule(test_tap2subunit))
     result.addTest(loader.loadTestsFromModule(test_filter_to_disk))
     result.addTest(loader.loadTestsFromModule(test_subunit_filter))
